src/engine/sparqlExpressions/StringExpressions.cpp

```diff
@@ -452,4 +474,12 @@ Expr makeEncodeForUriExpression(Expr child) {
   return make<EncodeForUriExpression>(child);
 }
 
-}  // namespace sparqlExpression
+Expr makeStrToIntExpression(Expr child) { 
+  return make<MakeStrToInt>(child); 
+}
+
+Expr makeStrToDoubleExpression(Expr child) {
+  return make<MakeDoubleToInt>(child);
+}
+
+}  // namespace sparqlExpression
```

Comment:
You have to setup clang-format anyway and run it:)

Reply:
How can i run it? - not sure if it worked